### PR TITLE
Hairenining minor

### PR DESCRIPTION
**Pull request by ProfessorPopoff**
This PR starts to unscuff hair when worn with a hat. I've painstakingly gone through each hair sprite and fitted it to the average hat. Everything should be good. Putting this up as a draft for now.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -622,8 +624,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				var/hair_file = S.icon
 				if(extensions[hair_state+dynamic_hair_suffix])
 					hair_state += dynamic_hair_suffix
-					hair_file = 'icons/mob/hair_extensions.dmi'
-
+				//MOJAVE EDIT BEGIN
+					hair_file = 'mojave/icons/mob/hair_extensions.dmi' //Original path is 'icons/mob/hair_extensions.dmi'
+				//MOJAVE EDIT BEGIN
```
Comment (TheRealScarHomie) [inline code/modules/mob/living/carbon/human/species.dm]: double begin :trollface:

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -611,7 +611,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				//List of all valid dynamic_hair_suffixes
 				var/static/list/extensions
 				if(!extensions)
-					var/icon/hair_extensions = icon('icons/mob/hair_extensions.dmi') //hehe
+				//MOJAVE EDIT BEGIN
+					var/icon/hair_extensions = icon('mojave/icons/mob/hair_extensions.dmi') //Original path is 'icons/mob/hair_extensions.dmi'
+				//MOJAVE EDIT BEGIN
```
Comment (Hekzder) [inline code/modules/mob/living/carbon/human/species.dm]:
```suggestion
				//MOJAVE EDIT END
```

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Hekzder) [reply code/modules/mob/living/carbon/human/species.dm]:
```suggestion
				//MOJAVE EDIT END
```

**Review by Hekzder — CHANGES_REQUESTED** [review]
Double `MOJAVE EDIT BEGIN` like Scar said.

**Comment by ProfessorPopoff** [comment]
> Double `MOJAVE EDIT BEGIN` like Scar said.

It's a feature.

**Comment by ProfessorPopoff** [comment]
There might be a few minor hiccups here and there, but overall every hair i've tested is pretty nice looking, now. 
e0052c5af46940a77f7f6b7b5eed3108
bbe9131fc3b0b31b6fe63c8a3b1a4c30
2e67fa4547ae873752c199625db7a145
03a13467f697f0e8d07b4f2b39af81f4

Ever hair will now not clip through the top of helmets/hat 100% guarantee, if not- your money back. They look pretty cool, now. Drip is at an all-time high. Opening this up, now.

**Review by Hekzder — APPROVED** [review]
Looks pretty good, tested it a bit and it seems to work decently well. I don't imagine this is 100% completely and utterly bug free but I wouldn't expect _every_ single headgear to have been tested on _every_ single hair style. 

Any potential related bugs are really tiny and we can squash them as they arise. Ultimately this is pretty epic. Massive drip buff.

**Pull request merged**